docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,167 @@
-- **Rail Terminal Fees**: Fees associated with the container at the rail terminal.
 
-### Integration Methods
+The following are new attributes that are specific to rail container tracking.
 
-There are two primary methods to integrate Terminal49’s rail tracking data programmatically: via API and DataSync.
+- **pod_rail_loaded_at**: Time when the container is loaded onto a railcar at the POD.
+- **pod_rail_departed_at**: Time when the container departs from the POD.
+- **inland_destination_eta_at**: Estimated Time of Arrival at the destination terminal.
+- **inland_destination_ata_at**: Actual Time of Arrival at the destination terminal.
+- **inland_destination_rail_unloaded_at**: Time when the container is unloaded from the rail terminal.
+- **pickup_facility_holds**: Holds placed on the container at the rail terminal.
+- **pickup_facility_fees**: Fees associated with the container at the rail terminal.
+- **pickup_facility_yard_location**: Yard location of the container at the rail terminal.
+- **pickup_facility_lfd_on**: Last Free Day for demurrage charges.
+- **pickup_rail_carrier_scac**: SCAC code of the rail carrier that picks up the container from the POD (this could be different than the rail carrier that delivers to the destination terminal). {/* Linear ticket to rename: https://linear.app/terminal49/issue/DATA-2767/rename-pickup-rail-carrier-scac */}
+{/* Linear ticket to add matching delivery/destination rail carrier scac: https://linear.app/terminal49/issue/DATA-2768/add-destination-rail-carrier-scac-to-serializer */}
 
-**A. Integration via API**
 
-Terminal49 provides a robust API that allows you to programmatically access rail container tracking data. Here’s a step-by-step guide to get started:
+{/* TODO: Look at the other container attributes that could be fed via rail but are currently shipping-line-only.  Such as :current_issues, :pickup_appointment_at, :availability_known, :available_for_pickup  */}
 
+These attributes can be found on [container objects](/api-docs/api-reference/containers/get-a-container).
 
-**B. Integration via DataSync**
+## Integration Methods
 
-Terminal49’s DataSync service provides a continuous feed of tracking data, automatically syncing with your system. Here’s how to set it up:
+There are two primary methods to integrate Terminal49's rail tracking data programmatically: via API and DataSync.
 
+### A. Integration via API
 
+Terminal49 provides a robust API that allows you to programmatically access rail container tracking data and receive updates via webhooks. You will receive rail events and attributes alongside events and attributes from the ocean terminal and carrier.
 
----
-Other Links 
-- [Blog post on North American intermodal rail visibility](https://www.terminal49.com/blog/launching-north-american-intermodal-rail-visibility-on-terminal49/).
+[Here's a step-by-step guide to get started](/api-docs/getting-started/start-here)
+
+
+### B. Integration via DataSync
```

Comment:
@SylvainGravejat is there a list of column names for the rail attributes that go into datasync?